### ISSUE TITLE
[Bug] Dark Cherry Trees are Missing Fuzzy Leaf Textures

### ISSUE THREAD

**Issue by dandy-is-lion** [satisfyu/Let-s-Do-Hub#116]
### Minecraft Version

1.20.1

### Mod

Vinery

### Mod Version

1.4.26

### Latest Version

- [X] Yes. I'm using the latest Version.

### Issue Description

Apple Trees display the correct fuzzy leaf textures, but Dark Cherry Trees do not. I recall this working in earlier versions. Tested with no other texture packs and with just Vinery installed (no aesthetic enhancing mods). Also, some suggestions while I'm at it: Apple Trees occasionally drop their apples from their leaves and Cherry Trees have tall variants like the Apple Trees.

![2024-07-17_06 45 56](https://github.com/user-attachments/assets/e39c4aaa-4464-4f47-a561-b04c0f6f3392)

With an aesthetic enhancing mod like [Immersive Weathering](https://modrinth.com/mod/immersive-weathering), and the fuzzy leaf textures enabled, Dark Cherry leaf piles appear glitched but Apple Tree leaf piles appear correct. When the texture pack is disabled, they both appear glitched. Not a big deal though as I can just blacklist these leaves. Just an aside related to the issue.

![image](https://github.com/user-attachments/assets/308a0ffb-51f1-462e-87e5-3fd51cb78a9f)
![image](https://github.com/user-attachments/assets/f366a940-ff10-4725-ac22-877e60462b35)

### Steps to Reproduce

1. Open Resource Packs
2. Enable "vinery/bushy_leaves"

### Minimal Instance Test

- [X] Yes, I've tested on a minimal instance.

### Check for Similar Issues

- [X] Yes, I have checked for similar issues.

**Comment by satisfyu** [comment]
This will be fixed in 1.4.27 - thank you. :) 

Apple Trees already have a small chance to drop apples.

**Comment by dandy-is-lion** [comment]
Looks awesome, thanks! Semi-related to this, I believe the Apple Tree and Dark Cherry saplings are missing vanilla sapling tags. Noticed this when filtering them with Create. I'm not familiar with modding but looking at Biomes O' Plenty, it seems you just need to add a file to `data/minecraft/tags/(blocks, items)/saplings.json` with an array of saplings in your mod.

Something like:
```
{
	"replace": false,
	"values": [
		"vinery:dark_cherry_sapling",
		"vinery:apple_tree_sapling"
	]
}
```

Though I've not tested this.
